pywt/src/_pywt.pyx

```diff
@@ -1068,7 +1068,8 @@ def _downcoef(part, np.ndarray[data_t, ndim=1, mode="c"] data,
             raise RuntimeError("Invalid output length.")
         coeffs = np.zeros(output_len, dtype=data.dtype)
 
-        if do_dec_a:
+        # for level > 1 and do_dec_a=False, always call dec_a until last level
```

Comment:
okay, I will update the comment here.  It is needed to mirror the behaviour of wavedec where successive level's detail coefficients correspond to a downsampling filtered version of the approximation coefficients at the next higher resolution level.